src/webview/static/base.css

```diff
@@ -569,6 +569,14 @@ hr {
   border-top: 1px solid hsla(0, 0%, 50%, 0.5);
 }
 
+h1, h2, h3, h4, h5, h6 {
+  font-weight: bold;
+  font-size: 100%;
+  text-decoration: underline;
+  margin-top: 0;
+  margin-bottom: 0.5rem;
+}
```

Comment:
Since these aren't identical to the web app's corresponding styles—

(from static/styles/rendered_markdown.css)
```css
{
  font-size: 14px;
  font-weight: 600;
  line-height: 1.4;
  margin-top: 0;
  margin-bottom: 5px;
  text-decoration: underline;
}
```

—we could put a brief comment saying what they do. That way, the default interpretation for this group of styles is that the differences aren't there to make a dramatic departure from the web app. Maybe a comment like "All heading levels have the same style, like in the web app."? It looks like https://github.com/zulip/zulip/issues/11418#issuecomment-459573917 is why the web app makes the slightly surprising choice of not differentiating the styles between h1, h2, etc., at all.

I think I don't mind leaving out comments on each individual difference from the web app, as long as we've made a reasonable effort to match it closely, with necessary changes so it works on mobile. Could, e.g., reorder them to be in the same order the web app uses. When someone revisits these particular styles, they'll likely have the corresponding web app code open at the same time, for comparison.

Reply:
Hmm, or maybe the section header is strong enough—

```js
/*
 Message contents.

 For the body of a message, we use nearly the same HTML as the webapp does.
 The styles therefore have a lot in common.  For the webapp's version of
 these styles, compare static/styles/rendered_markdown.css .
 */
```

—and the thing to do is mark any surprising/dramatic departures, and leave the boring differences unmarked.